File: 1/problem.ts
import { isMultipleOf } from "../common/common";

let accumulate = 0;
for (let j = 1; j < 1000; j++) {
    if (isMultipleOf(j, 3) || isMultipleOf(j, 5)) {
        accumulate += j;
    }
}

console.log("The sum of the first 1000 multiples of 5 or 3 is:", accumulate);

File: archive/1/problem.py
import math

def isMultipleof(x,y):
    if x % y == 0:
        return True
    else:
        return False

accumulate = 0
for j in range(1,1000):
    if isMultipleof(j,3) or isMultipleof(j,5):
        accumulate = accumulate + j

print("The sum of the first 1000 multiples of 5 or 3 is:", accumulate)